cylc/flow/tui/data.py

```diff
@@ -130,6 +130,46 @@
 }
 
 
+def cli_cmd(*cmd):
+    """Issue a CLI command.
+
+    Args:
+        cmd:
+            The command without the 'cylc' prefix'.
+
+    Rasies:
+        ClientError:
+            In the event of mishap for consistency with the network
+            client alternative.
+
+    """
+    proc = Popen(  # nosec (command constructed internally, no untrusted input)
+        ['cylc', *cmd],
+        stderr=PIPE,
+        stdout=PIPE,
+        text=True,
+    )
+    out, err = proc.communicate()
+    if proc.returncode != 0:
+        raise ClientError(f'Error in command {" ".join(cmd)}\n{err}')
```

Comment:
What do you think of
```suggestion
        raise ClientError(f'Error in command {" ".join(cmd)}\n{err or out}')
```

Reply:
I'm not sure the output viewer is up to scratch for displaying the full output (small narrow section of the window).